lib/slim_lint/linter/tab.rb

```diff
@@ -0,0 +1,17 @@
+module SlimLint
+  # Searches for tab indentation
+  class Linter::Tab < Linter
+    include LinterRegistry
+
+    MSG = 'Tab detected'.freeze
+
+    on_start do |_sexp|
+      dummy_node = Struct.new(:line)
+      document.source_lines.each_with_index do |line, index|
+        next unless line =~ /^( *)[\t ]*\t/
```

Comment:
The regex in use here was borrowed from the [rubocop tab detector](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/tab.rb#L16). It will detect lines that start with tabs or contain mixed tabs and spaces.